temporalcli/commands.workflow_history.go

```diff
@@ -0,0 +1,104 @@
+package temporalcli
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+
+	historypb "go.temporal.io/api/history/v1"
+	"go.temporal.io/sdk/client"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/encoding/prototext"
+)
+
+const (
+	historyFormatJSON      = "json"
+	historyFormatPB        = "pb"
+	historyFormatPBTXT     = "pbtxt"
+	historyFormatProto     = "proto"
+	historyFormatProtoText = "prototext"
+)
+
+func (c *TemporalWorkflowHistoryConvertCommand) run(cctx *CommandContext, args []string) error {
+	raw, err := os.ReadFile(c.SourceFile)
+	if err != nil {
+		return err
+	}
+
+	var history *historypb.History
+	switch c.SourceFormat.Value {
+	case historyFormatJSON:
+		history, err = client.HistoryFromJSON(bytes.NewReader(raw), client.HistoryJSONOptions{})
+		if err != nil {
+			return err
+		}
+
+	case historyFormatProtoText:
+		fallthrough
+	case historyFormatPBTXT:
```

Comment:
```suggestion
	case historyFormatProtoText, historyFormatPBTXT:
```

(and for other fallthrough cases in the PR)

Reply:
Done (in the one remaining use of fallthrough).

I kind of got into the habit of liberal use of fallthrough because I'm always thinking ahead to "when I have to convert this from switch-on-value to switch-on-first-matching-expression..."